docs/docs/10-reference.md

```diff
@@ -149,6 +149,7 @@ Options:
   - Snowpack uses Rollup internally to install your packages. This `rollup` config option gives you deeper control over the internal rollup configuration that we use.
   - **`installOptions.rollup.plugins`** - Specify [Custom Rollup plugins](#installing-non-js-packages) if you are dealing with non-standard files.
   - **`installOptions.rollup.dedupe`** - If needed, deduplicate multiple versions/copies of a packages to a single one. This helps prevent issues with some packages when multiple versions are installed from your node_modules tree. See [rollup-plugin-node-resolve](https://github.com/rollup/plugins/tree/master/packages/node-resolve#usage) for more documentation.
+  - **`installOptions.rollup.context`** - Specify top-level `this` value.
```

Comment:
Can you add another line of info here? Something like: "Specify top-level `this` value. Useful to silence install errors caused by legacy Common.js packages that reference a top-level `this` variable, which does not exist in a pure ESM environment."